packages/components/package.json

```diff
@@ -23,16 +26,30 @@
         "typecheck": "tsc --noEmit",
         "test": "vitest run --coverage",
         "test:ui": "vitest --ui",
-        "test:watch": "vitest"
+        "test:watch": "vitest",
+        "storybook": "storybook dev -p 6007",
+        "storybook:background": "storybook dev --no-open -p 6007",
```

Comment:
```suggestion
        "storybook:watch": "storybook dev --no-open -p 6007",
```

Reply:
why do you believe this should be called storybook:watch?

The difference from the normal storybook command is only that it is not triggering a browser to open

Reply:
obsolete as the script was removed